lib/gemstash/cli/setup.rb

```diff
@@ -122,6 +123,11 @@ def ask_timeout
         @config[:fetch_timeout] = timeout.to_i
       end
 
+      def ask_bachend
+        say_current_config(:backend, "Current backend")
+        @config[:backend] = ask_with_default("What backend?", %w[local s3], "local")
```

Comment:
Perhaps: Only keep `local` in the list of available backends, for now?

Reply:
I'm working on implementing the trigger actions for each backend including s3.
As a part of the process authorization to s3 is implemented.
So is it OK to keep both 'local' and 's3'?

Reply:
Right, I take it that this PR will contain the whole change, including support for both of the named backends.

Totally OK to keep both.

Thanks for sharing your plans, it's a good feature.